go-sbot build failure

# go.cryptoscope.co/ssb/cmd/go-sbot
cmd/go-sbot/main.go:172:15: first argument to append must be slice; have map[string]*ssb.KeyPair
cmd/go-sbot/main.go:174:51: cannot use kps (type map[string]*ssb.KeyPair) as type []*ssb.KeyPair in argument to multilogs.NewPrivateRea
d

--- a/cmd/go-sbot/main.go
+++ b/cmd/go-sbot/main.go
@@ -164,8 +164,13 @@ func main() {
 	if flagDecryptPrivate {
 		// TODO: refactor into plugins2
 		r := repo.New(repoDir)
-		kps, err := repo.AllKeyPairs(r)
+		kpsByPath, err := repo.AllKeyPairs(r)
 		checkFatal(errors.Wrap(err, "sbot: failed to open all keypairs in repo"))
+
+		var kps []*ssb.KeyPair
+		for _, v := range kpsByPath {
+		  kps = append(kps, v)
+		}
 
 		defKP, err := repo.DefaultKeyPair(r)
 		checkFatal(errors.Wrap(err, "sbot: failed to open default keypair"))

